v3/fleet/control-proxy.service

```diff
@@ -0,0 +1,31 @@
+[Unit]
+Description=run apigateway as control proxy
+Requires=docker.service
+After=docker.service bootstrap.service zk-health.service
+
+[Service]
+Environment="IMAGE=etcdctl get /images/control-proxy"
+Environment="INTERNAL_CONTROL_ELB=etcdctl get /images/control-proxy"
```

Comment:
@iderdik yes, you can reference that, as long as you set your `EnvironmentFile` above